.circleci/win-ram-disk.ps1

```diff
@@ -27,4 +27,4 @@ if (-not (Test-Path -Path $aimContents)) {
 
 # Setup RAM disk mount. Same parameters as ImDisk
 # See: https://support.circleci.com/hc/en-us/articles/4411520952091-Create-a-windows-RAM-disk
-./aim/cli/x64/aim_ll.exe -a -s 5G -m X: -p "/fs:ntfs /q /y"
+./aim/cli/x64/aim_ll.exe -a -s 12G -m X: -p "/fs:ntfs /q /y"
```

Comment:
Why such an increase in disk space usage for projects?

Reply:
bazel is running 6 tests concurrently.

I could try decreasing this to the minimum and/or reducing the concurrency if preffered?

Reply:
I am okay with this.